sdk/spring/spring-cloud-azure-starter-monitor/src/main/resources/META-INF/native-image/native-image.properties

```diff
@@ -1,7 +1,5 @@
 Args=\
-  --initialize-at-build-time=ch.qos.logback.core \
-  --initialize-at-build-time=ch.qos.logback.classic \
-  --initialize-at-build-time=io.opentelemetry.instrumentation.logback.appender.v1_0.internal.LoggingEventMapper \
-  --initialize-at-build-time=io.opentelemetry.instrumentation.api.internal.cache.concurrentlinkedhashmap.ConcurrentLinkedHashMap \
-  --initialize-at-build-time=io.opentelemetry.api.DefaultOpenTelemetry \
-  --initialize-at-build-time=io.opentelemetry.context.propagation.DefaultContextPropagators
+    --initialize-at-build-time=ch.qos.logback \
```

Comment:
Seems related to https://github.com/Azure/azure-sdk-for-java/blob/b555039bee5175240bcf6811f94a530dab4c6f5d/sdk/clientcore/core/src/main/resources/META-INF/native-image.io.clientcore.core/native-image.properties and https://github.com/Azure/azure-sdk-for-java/blob/b555039bee5175240bcf6811f94a530dab4c6f5d/sdk/core/azure-core/src/main/resources/META-INF/native-image/com.azure/azure-core/native-image.properties Logback build time initializations and the consequences of the `strict-image-heap` mode enabled by default from GraalVM 22.